server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -915,6 +920,13 @@ public int getNumberOfReplicas() {
         return settings.getAsInt(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, null);
     }
 
+    /**
+     * Returns true if segment replication is enabled on the index.
+     */
+    public boolean isSegrepEnabled() {
+        return Boolean.TRUE.equals(isSegRepEnabled);
```

Comment:
sorry, I'll stop commenting until you move this out of draft :)

Reply:
The logic in this method has changed with the latest commit - `IndexSettings` now stores the replication type itself rather than storing a boolean.

That said, I do like `SegRep` better than `Segrep` so i'll change the method name 😏